python/src/greenbids/tailor/core/app/tasks.py

```diff
@@ -0,0 +1,127 @@
+"""Repeated Tasks: Easily trigger periodic tasks on server startup.
+Copied from [fastapi_utils 0.7.0](https://github.com/dmontagu/fastapi-utils/blob/0.7.0/fastapi_utils/tasks.py)
+MIT Licence - [David Montague](https://github.com/dmontagu)
+"""
```

Comment:
question: Why the decorator code is copied if we can just install `fastapi-utils` and reuse `repeat_every` from it ?
https://github.com/dmontagu/fastapi-utils/blob/0.7.0/fastapi_utils/tasks.py

Reply:
Because `fastapi_utils` comes with many more features, and maybe some dependencies, etc.
Adding a dependency to a project have a cost and sometimes it's easier to just get what you need instead of the whole package.
Here, the file has no external dependency, is short and readable, so I feel it's easier to just copy it and avoid third party dependency.